GUI/Main.cs

```diff
@@ -453,7 +427,7 @@ public void UpdateCKAN()
             SwitchEnabledState();
             ClearLog();
             tabController.RenameTab("WaitTabPage", "Updating CKAN");
-            SetDescription("Upgrading CKAN to " + AutoUpdate.Instance.LatestVersion);
+            SetDescription("Upgrading CKAN to {AutoUpdate.Instance.LatestVersion}");
```

Comment:
Missing the dollar sign before the staring quote, I believe this will print the literal curly braces as text.

Reply:
How did VS screw that up?